asv_bench/benchmarks/io/csv.py

```diff
@@ -272,13 +271,12 @@ def setup(self, do_cache):
         self.StringIO_input = StringIO(data)
 
     def time_read_csv_cached(self, do_cache):
-        # kwds setting here is used to avoid breaking tests in
-        # previous version of pandas, because this is api changes
-        kwds = {}
-        if 'cache_dates' in _parser_defaults:
-            kwds['cache_dates'] = do_cache
-        read_csv(self.data(self.StringIO_input), header=None,
-                 parse_dates=[0], **kwds)
+        try:
```

Comment:
@TomAugspurger ok method of handling ?

Reply:
Seems fine.

Reply:
Although... I worry it would incorrectly catch a TypeError in the function? The other way might be to check `pandas.__version__`?

Reply:
hmm, let me see what i can do